packages/react-day-picker/test/po/po.ts

```diff
@@ -121,6 +121,18 @@ export function pressArrowDown() {
   userEvent.type(getFocusedElement(), '{arrowdown}');
 }
 
+export function pressTab() {
+  userEvent.tab();
+}
+
+export function pressShiftTab() {
+  userEvent.tab({ shift: true });
+}
+
+export function pressEnter() {
+  userEvent.keyboard('{enter}');
+}
+
 export function pressPageUp() {
   fireEvent.keyDown(getFocusedElement(), { key: 'PageUp', charCode: 33 });
```

Comment:
I really like the userEvent API! I think for our purposes, `userEvent.keyboard` should be used instead of `fireEvent.keyDown` and instead of `userEvent.type` (used for text inputs or text areas).

https://testing-library.com/docs/ecosystem-user-event/#typeelement-text-options

Reply:
I have half a mind to change it here, but it seems like scope creep, so I'll contain myself.

Reply:
Make sense thanks. https://github.com/gpbl/react-day-picker/issues/1348